compose/ui/ui/src/skikoMain/kotlin/androidx/compose/ui/scene/ComposeSceneLayer.skiko.kt

```diff
@@ -159,6 +171,7 @@ internal fun rememberComposeSceneLayer(
     val density = LocalDensity.current
     val layoutDirection = LocalLayoutDirection.current
     val parentComposition = rememberCompositionContext()
+    val compositionLocalContext by rememberUpdatedState(currentCompositionLocalContext)
```

Comment:
It's already contained inside `parentComposition` that we already passing, but it's internal to runtime, so we cannot receive it from there.
At the same time, we must create composition for layers since it has different scope/invalidations/etc

 So for now we have to pass it separately 😞